Dockerfile

```diff
@@ -73,8 +73,8 @@ ARG DGL_BACKEND=pytorch
 ENV DGL_BACKEND=$DGL_BACKEND
 ENV DGLBACKEND=$DGL_BACKEND
 
-RUN pip install --no-cache-dir --no-deps dgl -f https://data.dgl.ai/wheels/cu121/repo.html
-RUN pip install --no-cache-dir --no-deps dglgo -f https://data.dgl.ai/wheels-test/repo.html
+RUN pip install --no-cache-dir --no-deps dgl==2.0.0 -f https://data.dgl.ai/wheels/cu121/repo.html
+RUN pip install --no-cache-dir --no-deps dglgo==2.0.0 -f https://data.dgl.ai/wheels-test/repo.html
```

Comment:
dglgo is not 2.0.0, I dont know why we even install this. Its max version is 0.2.0

Reply:
I only know this because I did the same thing

https://pypi.org/project/dglgo/

Reply:
Yea, good catch! I removed the version specified from `dglgo`. As far as removing `dglgo` altogether is considered, that is fair. As I understand that's just a CLI interface for DGL. Let me check if we can completely remove it.

Reply:
Removed DGLGO